include/kamping/collectives/neighborhood/alltoall.hpp

```diff
@@ -43,14 +43,16 @@
 
 /// @brief Wrapper for \c MPI_Neighbor_alltoall.
 ///
-/// This wrapper for \c MPI_Neighbor_alltoall sends the same amount of data from a rank i to each of its neighbour j for
-/// which an edge (i,j) in the communication graph exists. The following buffers are required:
+/// @todo check again once the concrete semantics (potential differing number of send/recv counts) of
```

Comment:
Add the todo as a separate line to the bottom, for making the rendered docs look better.